modules/foreman_ptable.py

```diff
@@ -241,10 +241,12 @@
 def sanitize_ptable_dict(ptable_dict):
     # This is the only true source for names (and conversions thereof)
     name_map = {
+        'template': 'layout', # the parse_template_* methods stores the "layout" in "template"
         'layout': 'layout',
         'locations': 'location',
         'name': 'name',
         'organizations': 'organization',
+        'oses': 'os_family', # the foreman community templates are using oses instead of os_family (which is wrong?)
```

Comment:
Yes, I think `os_family` agrees with [the API](https://github.com/theforeman/foreman/blob/develop/app/controllers/concerns/foreman/controller/parameters/ptable.rb#L11)